packages/app/src/app/components/SignInModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useCallback } from 'react';
+import { ThemeProvider, Stack } from '@codesandbox/components';
+import codeSandboxBlack from '@codesandbox/components/lib/themes/codesandbox-black';
+import OutsideClickHandler from 'react-outside-click-handler';
```

Comment:
Do we already have this in the `package.json` of `app`? 🤔

Reply:
It is there as far as I can tell?